test/tests.js

```diff
@@ -29127,3 +29127,141 @@ test("/[a-z]/gim", {
 testFail("/[a-z]/u", "Invalid regular expression flag (1:1)");
 testFail("/[a-z]/y", "Invalid regular expression flag (1:1)");
 testFail("/[a-z]/s", "Invalid regular expression flag (1:1)");
+
+// https://github.com/ternjs/acorn/issues/255
+
+test("(a) = {}", {
```

Comment:
Just recalled what I was more curious about - nested patterns like `({ src: (dest) } = obj)` and other combinations. Do they work / can you please add tests for them?